src/components/button/button.component.js

```diff
@@ -16,6 +16,7 @@ function renderChildren({
   children,
   disabled,
   buttonType,
+  tooltipMessage,
```

Comment:
Might be worth surfacing `tooltipPosition` as well here

Reply:
✅